libs/common-transfers/src/lib/components/excluded-dates.component.ts

```diff
@@ -23,17 +25,9 @@ export class ExcludedDatesComponent implements PaymentFormField, OnInit, OnDestr
   control!: AbstractControl;
   validationControl!: FormControl;
 
-  validationLabel = 'Start on';
+  validationLabel = 'Execution date';
   validationMessages = [
-    { name: 'excludedDate', message: 'Cannot transfer on this day' },
-    {
-      name: 'pastDate',
-      message: 'Start date cannot be in the past',
-    },
-    {
-      name: 'futureDate',
-      message: 'Start date cannot be past this date',
-    },
+    { name: 'excludedDate', message: 'Cannot transfer on this day' }
```

Comment:
We should provide a default message since this is a custom validator and there is none in the product